docs/data/install-hadoop-2-0-0.md

```diff
@@ -0,0 +1,1499 @@
+| `hdfs`    | Used by Hadoop to store data blocks and meta-data |
+
+For this package to function correctly, you will have to create the users needed for grid operation. Any user that can be authenticated should be created.
+
+For grid-mapfile users, each line of the grid-mapfile is a certificate/user pair. Each user in this file should be created on the server.
+
+For gums users, this means that each user that can be authenticated by gums should be created on the server.
+
+Note that these users must be kept in sync with the authentication method. For instance, if new users or rules are added in gums, then new users should also be added here.
+
+Certificates
+------------
+
+<span class="twiki-macro STARTSECTION">Certificates</span>
+
+| Certificate                 | User that owns certificate | Path to certificate                                                                                 |
+|:----------------------------|:---------------------------|:----------------------------------------------------------------------------------------------------|
+| Host certificate            | `root`                     | `/etc/grid-security/hostcert.pem` <br> `/etc/grid-security/hostkey.pem`                       |
+| Bestman service certificate | `bestman`                  | `/etc/grid-security/bestman/bestmancert.pem` <br> `/etc/grid-security/bestman/bestmankey.pem` |
+
+<span class="twiki-macro ENDSECTION">Certificates</span>
+
+[Instructions](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallCertScripts) to request a service certificate.
+
+You will also need a copy of CA certificates (see below). Note that the `osg-se-hadoop-srm` and `osg-se-hadoop-gridftp` package will automatically install a certificate package but will not necessarily pick the cert package you expect. For instance, certain installs will prefer the `osg-ca-scripts` package to fulfill this requirement, which installs a set of scripts to automatically update the certificates, but does not initialize the CA certs by default (you have to run it first). For this reason, you may want to specifically install the cert package of your choice first, before installing Hadoop.
+
+Networking
+----------
+
+For more details on overall Firewall configuration, please see our [Firewall documentation](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/FirewallInformation).
+
+**NOTE:** The versions of Hadoop in OSG series 3.1 and 3.2 (ie, Hadoop 0.20 and 2.0.0) do not inter-operate. In order to use Hadoop 2.0.0, *all* nodes in the hadoop system (namenode, secondary namenode, datanodes, srm/gridftp nodes and all client nodes) must update to OSG 3.2 and Hadoop 2.0.0.
+
+Initializing Certificate Authority
+==================================
+
+This is needed by GridFTP and SRM nodes, but it is recommended for all nodes in the cluster.
+
+++ Enable and Start `fetch-crl` To enable fetch-crl (fetch Certificate Revocation Lists) services by default on the node:
```

Comment:
Why reproduce the entirety of https://opensciencegrid.github.io/docs/common/ca/#install-fetch-crl when you can just link it?